app/components/subcases/subcase-description/template.hbs

```diff
@@ -48,62 +44,60 @@
           {{t "number-session"}}
         </span>
         <span class="vl-description-data__value">
-          {{#if (not (await subcase.isOC))}}
+          {{#if (await subcase.case.latestMeeting.number)}}
+            {{await subcase.case.latestMeeting.number}}
+          {{else}}
```

Comment:
I can't seem to figure out where this comes from?

Reply:
this span used to be 
```
        <span class="vl-description-data__value">
          {{#if (not (await subcase.isOC))}}
            {{t "no-number-yet"}}
          {{else}}
            {{await subcase.case.OCMeetingNumber}}
          {{/if}}
        </span>
```

-subcase.isOC relation has been removed
-OCMeetingNumber was a computed value that was equal to `subcase.case.latestMeeting.number`
The computed was removed but the number value (if present) should still be shown on the subcase description view